master/getting-started/kubernetes/upgrade/delete.md

```diff
@@ -104,4 +104,50 @@ This procedure requires etcdctl v3. The etcdctl tool is installed along with etc
 ### Next steps
 
 Return to [Migrate your data](/{{page.version}}/getting-started/kubernetes/upgrade/migrate)
-to try again.
\ No newline at end of file
+to try again.
+
+## Deleting Calico data from the Kubernetes API datastore after a downgrade
```

Comment:
Need to add a bullet around line 17 explaining when you might need to execute this procedure and linking to the anchor link here (like the other bullets)